lightning/src/ln/peer_handler.rs

```diff
@@ -234,6 +234,13 @@ enum InitSyncTracker{
 	NodesSyncing(PublicKey),
 }
 
+/// When the outbound buffer has this many messages, we'll stop reading bytes from the peer until
+/// we manage to send messages until we reach this limit.
+const OUTBOUND_BUFFER_LIMIT_READ_PAUSE: usize = 10;
```

Comment:
Shouldn't this constant call OUTBOUND_BUFFER_LIMIT_SYNC_PAUSE ? Otherwise, it's a bit confusing given it's called in `do_attempt_write_data`

Reply:
Hmm, its used for both, not sure which is better to focus on. I could just call it like `OUTBOUND_BUFFER_LOW_WATER_MARK` which is pretty normal for these types of constants, and document in the doccomment what its used for?